processor/configuration/src/main/resources/schemas/sources/azure_eventhubs_source_0.1.json

```diff
@@ -0,0 +1,69 @@
+{
+  "type" : "object",
+  "additionalProperties" : false,
+  "required" : [
+    "azure_namespace_name",
+    "azure_eventhub_name",
+    "azure_shared_access_name",
+    "azure_shared_access_key",
+    "azure_blob_account_name",
+    "azure_blob_access_key",
+    "azure_blob_container_name"
+  ],
+  "properties" : {
+    "azure_namespace_name" : {
+      "title" : "Eventhubs Namespace",
+      "description" : "The Event Hubs namespace.",
+      "type" : "string"
+    },
+    "azure_eventhub_name" : {
+      "title" : "Eventhubs Name",
+      "description" : "The Event Hub name.",
+      "type" : "string"
+    },
+    "azure_shared_access_name" : {
+      "title" : "Share Access Name",
+      "description" : "The Event Hubs SAS key name.",
+      "type" : "string",
+      "x-group" : "credentials"
+    },
+    "azure_shared_access_key" : {
+      "title" : "Share Access Key",
+      "x-group" : "credentials",
+      "oneOf" : [ {
+        "title" : "Share Access Key",
+        "description" : "The key for the EventHubs SAS key name.",
```

Comment:
Reading other description fields, shouldn't we use `Event Hubs` instead of `EventHubs`?

Reply:
same as before.